package.json

```diff
@@ -139,7 +146,7 @@
 		"pretty-quick": "^3.1.0",
 		"ts-jest": "^26.5.4",
 		"ts-node": "^9.1.1",
-		"typescript": "4.2.3"
+		"typescript": "4.2.4"
```

Comment:
Revert lest this is going to cause conflicts down the road, i.e. if not merged before sunday and depfu comes around.